com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1636,50 +1682,67 @@ def _root_from_endpoint(endpoint):
         return resource_url, name
 
     @staticmethod
-    def of_endpoint(endpoint=None, service_name=None, username=None, password=None, verify=None):
+    def of_endpoint(endpoint=None, service_name=None, username=None, password=None, verify=None, rest_endpoint=None, security_endpoint=None):
```

Comment:
Is it possible to just have `endpoint` rather than `endpoint` and `rest_endpoint`.

Could the presence of `security_endpoint` indicate a stand-alone configuration?

Reply:
It would be possible to remove `rest_endpoint` and use `endpoint` instead.  I think it may be less confusing to have two different parameters, rather than a single parameter that is specified differently depending on whether it is a stand-alone or integrated configuration.

Reply:
However, currently `service_name` must be specified.  In a stand-alone configuration, is there always exactly one instance?  If so, we don't actually need to know the instance name, otherwise we do.